### PR TITLE
 Fix toasts fonts 

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] #1202
[Problem] Toast message has incorrrect fonts
[Solution] Apply OneUI 2.2 fonts

### CODE REVIEW

**Review by DohyungLim — COMMENTED** [review]
I think have not problem.

File: src/css/profile/mobile/theme-changeable/theme.font.less [hunk 1]
```diff
@@ -117,10 +117,10 @@
 @font_size_ticker_text2: 30 * @unit_base;		// T0941: Ticker noti text2
 
 //**************************************************************************
-//                   Smallpopup
+//                   Toast
 //**************************************************************************
-@font_size_toast_text: 30 * @unit_base;		// T1221: Toast text
-@font_size_toast_button_text: 30 * @unit_base;		// T1222: Toast button text
+@font_size_toast_text: 16 * @px_base;		// T1221: Toast text
+@font_size_toast_button_text: 18 * @px_base;		// T1222: Toast button text
```
Comment (HunseopJeong) [inline src/css/profile/mobile/theme-changeable/theme.font.less]: Use @sp_base for the text.

Comment (lslachciak) [reply src/css/profile/mobile/theme-changeable/theme.font.less]: fixed

**Comment by lslachciak** [comment]
Improved PR according to the @HunseopJeong comment

**Review by HunseopJeong — APPROVED** [review]
LGTM

**Pull request merged**